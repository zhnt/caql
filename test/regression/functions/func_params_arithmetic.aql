function multiply(x, y) {
    let result = x * y
    print("multiply(" + string(x) + ", " + string(y) + ") = " + string(result))
    return result
}

function divide(a, b) {
    let result = a / b
    print("divide(" + string(a) + ", " + string(b) + ") = " + string(result))
    return result
}

multiply(6, 7)
divide(100, 5)