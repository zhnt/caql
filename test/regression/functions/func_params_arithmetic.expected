multiply(6, 7) = 42
divide(100, 5) = 20
